CakePHP/ruleset.xml

```diff
@@ -11,6 +11,26 @@
     <rule ref="PSR2" />
     <rule ref="PSR12" />
 
+    <!-- https://github.com/squizlabs/PHP_CodeSniffer/issues/2619 -->
+    <rule ref="PSR12.Files.FileHeader">
```

Comment:
This sniff is buggy, the spacing / block ordering rules it enforces are silly and unlikely to be auto fixable. Hence ignoring it. We can see how it works again once new phpcs release is done.